Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserExpandedTableView.h

```diff
@@ -33,11 +34,16 @@ namespace AzToolsFramework
         class ExpandedTableViewDelegate
             : public QStyledItemDelegate
         {
+            Q_OBJECT
         public:
             AZ_CLASS_ALLOCATOR(ExpandedTableViewDelegate, AZ::SystemAllocator);
             ExpandedTableViewDelegate(QWidget* parent = nullptr);
 
             void paint(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const override;
+            QWidget* createEditor(QWidget* parent, const QStyleOptionViewItem& option, const QModelIndex& index) const override;
+
+        signals:
+            void renameTableEntry(const QString& value) const;
```

Comment:
hmmm... I guess a signal is const, but we don't usually do this by convention. I'm surprised this compiles, since Qt's MOC is what actually generates this funciton, and I doubt it adds the const qualifier...

Reply:
Sounds good, will remove the const